applications/tari_console_wallet/src/automation/command_parser.rs

```diff
@@ -216,6 +218,21 @@ fn parse_send_tari(mut args: SplitWhitespace) -> Result<Vec<ParsedArgument>, Par
     Ok(parsed_args)
 }
 
+fn parse_list_utxos(mut args: SplitWhitespace) -> Result<Vec<ParsedArgument>, ParseError> {
+    let mut parsed_args = Vec::new();
+
+    // amount
+    if let Some(v) = args.next() {
+        if v == "csv_file" {
+            if let Some(v) = args.next() {
+                parsed_args.push(ParsedArgument::CSVFile(v.to_string()));
+            }
+        }
+    };
```

Comment:
This is a bit strange to put the csv_file literal as a command argument and it's looking more like we'll need a standard way to push command output to various streams (file, csv, stdout etc.)

For now perhaps `list-utxos csv:my-file.csv` or `list-utxos csv-file:my-file.csv`?

```suggestion
    let parse_args = args.next().and_then(|s| {
      let mut parts = s.split(":");
      // Return None if first part isn't "csv"
      parts.next().filter(|s| s == "csv")?;
      parts.next().map(|s|   ParsedArgument::CSVFile(v.to_string()))
     }).into_iter().collect();
```

Reply:
Personally I like this user format better:
`tari_console_wallet --command "export-utxos csv_file <file name>"`

Reply:
yeah LGTM - my point was mostly that we should maybe have a standard for these commands. I do think that `--csv-file` makes it clearer that csv_file is an option for the command 👍🏻